### DISCUSSION TITLE
"Add Torrent" API

### DISCUSSION THREAD

**Discussion by Roycinger** [Novik/ruTorrent#2612] [Q&A] [answered]
Hi folks,

I've gone through various plugins and the Wiki but didn't find I was looking for.
Is there a possibility to tripper an API from a local script (like Powershell) to add a local torrent file to the remote instance of ruTorrent?

Thanks in advance!

**Comment by stickz** [comment, marked as answer]
Yes, you can do whatever you want with xmlrpc. This is a remote control console. Sonarr or Radarr might have an example.

ruTorrent is a front end for rTorrent. You can expose the xmlrpc or in some cases use the httprpc plugin.

**Reply by Roycinger** [reply]
Thanks for getting back!
My instance of ruTorrent is hosted externally, I'm not sure if I'm allowed to install anything additionally.
I hoped there's an API out of the box.
But thanks, I'll read up on xmlrpc!
